engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/sampleTransformers.scala

```diff
@@ -157,10 +157,18 @@ object sampleTransformers {
                   )), defaultMode = DualEditorMode.SIMPLE) aggregator: Aggregator,
                 @ParamName("aggregateBy") aggregateBy: LazyParameter[AnyRef],
                 @ParamName("windowLength") length: java.time.Duration,
-                @ParamName("emitExtraWindowWhenNoData") emitExtraWindowWhenNoData: Boolean,
+                @ParamName("emitWhen")
+                @DualEditor(simpleEditor = new SimpleEditor(
+                  `type` = SimpleEditorType.FIXED_VALUES_EDITOR,
+                  possibleValues = Array(
+                    new LabeledExpression(label = "On each event", expression = "'OnEvent'"),
```

Comment:
WDYT about extracting java enum and use here this `T(...).VALUE` approach like we have in `aggregator`? It will be consistent + less code parsing this parameter.

Reply:
:+1: done